snooty.toml

```diff
@@ -14,6 +14,7 @@ toc_landing_pages = [
   "/authentication",
   "/authentication/providers",
   "/mongodb",
+  "/getting-started",
```

Comment:
To prevent the `tutorials` page from showing up as a landing page when you click its entry in the ToC, remove its entry from this landing pages array. That is - remove line 33/34 down below in this file where it says `"/tutorials",`

You will still see tutorials in the sidebar, because you define it in the `toctree` directive in the Getting Started file at line 12, but it will just be a container that opens instead of actually showing you a landing page.